pkg/virt-config/configuration_test.go

```diff
@@ -298,6 +298,42 @@ var _ = Describe("test configuration", func() {
 		),
 	)
 
+	var stage v1.VMRolloutStrategy = v1.VMRolloutStrategyStage
+	var liveUpdate v1.VMRolloutStrategy = v1.VMRolloutStrategyLiveUpdate
```

Comment:
Yeah but then the test entries have to use `(*v1.VMRolloutStrategy)(pointer.P(v1.VMRolloutStrategyLiveUpdate))` which is quite ugly. Otherwise ginkgo decides it's a string...
So I think having the 2 vars is just simpler, let me know if you disagree!